de-lib/src/main/java/org/iplantc/de/client/services/impl/AppUserServiceFacadeImpl.java

```diff
@@ -439,4 +440,18 @@ public void unshareApp(AppUnSharingRequestList request, DECallback<String> callb
         ServiceCallWrapper wrapper = new ServiceCallWrapper(POST, address, payload);
         deServiceFacade.getServiceData(wrapper, callback);
    }
+
+    @Override
+    public void isPublishable(String system_id, String id, DECallback<Boolean> callback) {
+        String address = APPS + "/" + system_id + "/" + id + "/is-publishable";
+        ServiceCallWrapper wrapper = new ServiceCallWrapper(address);
+        deServiceFacade.getServiceData(wrapper, new DECallbackConverter<String, Boolean>(callback) {
+            @Override
+            public Boolean convertFrom(String result) {
+                JSONObject obj = JsonUtil.getInstance().getObject(result);
```

Comment:
I think it would be better to use an `AutoBean` here instead of `JsonUtil`, especially since the `reason` field was just added by cyverse-de/apps#74